dashboard/app/models/user.rb

```diff
@@ -891,20 +891,14 @@ def recent_courses_and_scripts
     user_scripts = in_progress_and_completed_scripts.
       select {|user_script| !course_scripts_script_ids.include?(user_script.script_id)}
 
-    # TODO: replace courseName with name (here and in components) as some of these
-    # are courses and some are scripts
-    # TODO: may make more sense to generate links on the client (might need to
-    # provide both name and title to accomplish this). If we do we can get rid of
-    # include Rails.application.routes.url_helpers
-    # TODO: assigned sections seems like it belongs somewhere else
-    # TODO: currently no way for levelbuilders to edit description_short
-
     course_data = courses.map do |course|
       {
-        courseName: data_t_suffix('course.name', course[:name], 'title'),
+        name: data_t_suffix('course.name', course[:name], 'title'),
         description: data_t_suffix('course.name', course[:name], 'description_short'),
         link: course_path(course),
         image: '',
+        # assigned_sections is current unused. When we support this, I think it makes
+        # more sense to get/store this data separately from courses.
```

Comment:
The only reason I had assigned_sections here is because each individual course card needs to know whether there are sections assigned to it or not so it seems like information that goes with a course for that purpose.